api/schema.go

```diff
@@ -8,6 +8,22 @@ import (
 	"github.com/puppetlabs/wash/plugin"
 )
 
+// swagger:route GET /fs/schema schema entrySchema
+//
+// Schema for an entry at path
+//
+// Returns an EntrySchema object describing the plugin schema starting at the given path.
```

Comment:
Looking at some other types (e.g. `execBody/execResponse`), I think you can document this as `map[string]*EntrySchema` noting that the first key in the map corresponds to `<path>`'s schema.

Reply:
Ah, because that's how it'll be marshaled?

Reply:
And are those apitypes.EntrySchema or plugin.EntrySchema?

Reply:
Looks kind of challenging to document.